src/client/components/CourseReserves/__snapshots__/test.js.snap

```diff
@@ -2,22 +2,165 @@
 
 exports[`Correctly diplays alert when URL is none 1`] = `
 <div>
-  <Iframe />
+  <div>
+    <Alert
```

Comment:
@kilometerskang Is this autogenerated by the tests or something? Why is it repeated 3 times?

Reply:
These are the snapshots auto-generated by the Enzyme snapshot tests